.travis.yml

```diff
@@ -3,11 +3,24 @@ sudo: false
 python:
   - 2.6
   - 2.7
-  - 3.2
   - 3.3
+  - 3.4
+  - pypy
 
-script: python test.py
+install:
+  - travis_retry pip install python-coveralls nose-cov
+  - python setup.py install
```

Comment:
Any reason to prefer that over `python setup.py install` if we're using setuptools?